app/features/new-app-required/use-new-app-required.js

```diff
@@ -0,0 +1,83 @@
+import { useEffect, useState } from 'react';
+import { Platform } from 'react-native';
+
+const MINIMUM_UPDATEABLE_VERSION_NUMBER = 22;
+const FETCH_URL = 'https://api-staging.mindlogger.org/api/v1/applet/mobile-upgrade-info';
```

Comment:
No need to use this constant if you put the fetch request to network.js and reuse `get` function